tests/functional/tdtagged/td1dtagged.py

```diff
@@ -111,9 +152,11 @@ def vthz(x):
     ElectronModel(closure="isothermal", Te=0.12)
 
 
+    sim = ph.global_vars.sim
 
-    timestamps = all_timestamps(gv.sim)
-
+    dtdiag = sim.time_step
+    ndiag  = sim.final_time/dtdiag+1
+    timestamps = dtdiag * np.arange(ndiag)
```

Comment:
is this better?

Reply:
than?

Reply:
"timestamps = all_timestamps(gv.sim)"

Reply:
right! forgot about it probably. will change